File: app/src/main/java/com/example/disneycharacter/ItemElement.java
package com.example.disneycharacter;

public class ItemElement {
    private String name;
    private String content;
    private int icon;

    public ItemElement(String name,String content, int icon){
        this.name=name;
        this.content = content;
        this.icon=icon;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getIcon() {
        return this.icon;
    }
    public void setIcon(int icon) {
        this.icon = icon;
    }

    public String getContent() {
        return this.content;
    }

    public void setContent(String name) {
        this.content = content;
    }




}

File: app/src/main/java/com/example/disneycharacter/ItemElement.kt
package com.example.disneycharacter

class ItemElement {
    private var name:String = ""
    private var icon:Int

    constructor(name: String, icon: Int){
        this.name = name
        this.icon = icon
    }

    public fun getName(): String{
        return this.name
    }

    public fun getIcon(): Int{
        return this.icon
    }

    public fun setName(name: String){
        this.name = name
    }

    public fun setIcon(icon: Int){
        this.icon = icon
    }

}